siac/walletcmd.go

```diff
@@ -116,6 +109,42 @@ Run 'wallet send --help' to see a list of available units.`,
 		Long:  "View wallet status, including the current balance and number of addresses.",
 		Run:   wrap(walletstatuscmd),
 	}
+
+	walletUnlockCmd = &cobra.Command{
+		Use:   "unlock \"[password]\"",
+		Short: "Unlock the wallet",
+		Long:  "Decrypt and load the wallet into memory",
+		Run:   wrap(walletunlockcmd),
+	}
+
+	walletInitCmd = &cobra.Command{
+		Use:   "init",
+		Short: "Initilize and encrypt a new wallet",
+		Long: `Generate primary seed used for the wallet and encrypt using that
```

Comment:
meh, maybe a little bit. The fact that it returns the seed, and that the command takes no arguments, should enough of a hint.